frontend/src/components/Home/MainContent.js

```diff
@@ -1,13 +1,30 @@
 import React, { Component } from 'react';
 import JokeCard from "./JokeCard"
 import Grid from '@mui/material/Grid';
+import Button from '@mui/material/Button';
 import ItemSearchCard from './ItemSearchCard';
 
 class MainContent extends Component {
     constructor(props) {
         super(props);
     }
 
+    _createJokeCards() {
+        if (!this.props.jokes) return null;
+
+        return this.props.jokes.map(joke => {
+            return (
+                <Grid item xs={12} sm={4} key={joke.id}>
```

Comment:
It would be nice to align the heights:
![image](https://user-images.githubusercontent.com/10985546/165668520-79d3b425-376e-449d-b032-4b250dd7314d.png)

Reply:
Should be possible to align all items in a row, and yeah would look way better

Reply:
done.